Reference/V9-Config/GlobalSettings/index-v10.md

```diff
@@ -0,0 +1,261 @@
+
+### Install missing database
+
+This is not a setting that commonly needs to be configured.
+
+If enabled Umbraco will try to automatically install the database when it's missing. This is primarily used in conjuction with unattended installs.
+
+### Disable election for single server
+
+This is not a setting that commonly needs to be configured.
+
+This value is primarily used on Umbraco Cloud for a small startup performance optimization. When this is true, the website instance will automatically be configured to not support load balancing and the website instance will be configured to be the 'primary' server for scheduling so no [primary election](https://our.umbraco.com/documentation/Getting-Started/Setup/Server-Setup/load-balancing/file-system-replication.md) occurs. This will save 1 database call during startup.
+
+### Database factory version
+
+This is not a setting that commonly needs to be configured. 
+
+This setting is used to specify which sql server version that the database is running, this setting is only required if you use SqlServer 2008, if this is the case set the setting to `"SqlServer.V2008"`
+
+### Main dom lock
+
+Specifies the implementation of `IMainDomLock` to be used.
+
+IMainDomLock is used to synchronize access to various resources e.g. Lucene indexes.
+
+Available options:
+
++ **FileSystemMainDomLock** - Available cross-platform, uses lock files written to LocalTempPath to control acquisition of MainDom status.
++ **MainDomSemaphoreLock** - Windows only, uses a named Semaphore with a `maximumCount` of 1 to control acquisition of MainDom status.
++ **SqlMainDomLock** - Available cross-platform, uses the database to control acquisition of MainDom status.
+
+The default implementation unless configured otherwise is `FileSystemMainDomLock`.
+
+
+### Main dom key discriminator
+
+For advanced use cases e.g. deployment slot swapping on Azure app services.
+
+When using SqlMainDomLock a MainDomKey is used to identify an instance of a running application.
+
+The MainDomKey is by default comprised of the server's machine name & the application id.
+This is generally all that is required to control MainDom status as starting a new process for the same application on the same
+server will result in a matching MainDomKey, requiring that an existing instance yields MainDom status to the new process.
+
+Deployment slots for a given Azure app service share the same machine name and without additional configuration will share a
+MainDomKey and therefore compete for MainDom status, this can be undesirable if attempting to deploy to a deployment slot
+followed by a swap with the production slot as once traffic has switched to the new instance the old production instance reboots
+and can re-acquire MainDom status.
+See [What happens during a swap](https://docs.microsoft.com/en-us/azure/app-service/deploy-staging-slots#what-happens-during-a-swap).
```

Comment:
```suggestion
Deployment slots for a given Azure App Service share the same machine name. Without additional configuration, they will share a MainDomKey and therefore compete for MainDom status. This can be undesirable if attempting to deploy to a deployment slot followed by a swap with the production slot as once traffic has switched to the new instance the old production instance reboots and can re-acquire MainDom status.
See [What happens during a swap](https://docs.microsoft.com/en-us/azure/app-service/deploy-staging-slots#what-happens-during-a-swap).
```